common/src/main/java/com/redhat/cloud/notifications/models/NotificationStatus.java

```diff
@@ -0,0 +1,22 @@
+package com.redhat.cloud.notifications.models;
+
+public enum NotificationStatus {
+    // The notification failed to be created for processing
+    // This could happen when sending the notification to a different processor and it was rejected before entering a processing phase
+    FAILED_CREATION,
```

Comment:
I wonder if instead of `FAILED_CREATION`/`FAILED_PROCESSING` we should have:

- `FAILED_INTERNAL` for any exception that is thrown in Notifications, including when we fail to send an event to RHOSE. This would tell us that we can find an explanation in our logs/metrics.
- `FAILED_EXTERNAL` for errors returned by Camel or RHOSE.

Reply:
It sounds good